### ISSUE TITLE
Theme: Update Info Text for Website & Add Info Text for Tagline

### ISSUE THREAD

**Issue by eaquigley** [IQSS/dataverse#1629]
Update the info/help text above the Website field. Also need to add info/help text about the tagline only being 140 characters.

**Comment by eaquigley** [comment]
Passing to @mheppler 

Help text for Tagline: Provide a tagline that is 140 characters or less. 

Error Message for when Tagline isn't filled out but Website is: Please enter a tagline for the website to be hyperlinked with.

**Comment by mheppler** [comment]
Old text, to search for in backing bean: "Tagline is required for Website."

**Comment by mheppler** [comment]
Added new help text to Theme + Widget page, and new validation error message to backing bean.

Noticed that the help text for Website is a bit wordy, breaking to a second line, and nerdy, "hyperlink" rather than just "link" we use everywhere else on the page. Sending back to QA to decide if anything more needs to be done.

**Comment by eaquigley** [comment]
@mheppler changed from hyperlinked to linked. hopefully this will prevent it from being two lines now.

**Comment by eaquigley** [comment]
Help text appearing as it should. Closing ticket.
